Formula/mysql.rb

```diff
@@ -69,6 +69,12 @@ def install
     rm_rf lib/"plugin/libcrypto.1.1.dylib"
     rm_rf lib/"plugin/libssl.1.1.dylib"
 
+    # edit mysql_config to make sure anything that uses it to figure out
+    # what libraries to link looks for openssl in the right place
+    inreplace bin/"mysql_config" do |s|
+      s.gsub! "libs=\"-L$pkglibdir\"", "libs=\"-L$pkglibdir -L#{Formula["openssl@1.1"].lib}\""
+    end
```

Comment:
This is a bug in the MySQL code and needs at the very least a linked upstream bugreport and preferably have a patch submitted there.

Reply:
No, this is required because of homebrew's deletion of MySQL's own copies of the SSL libraries in the immediately preceding lines.

Reply:
Which in turn is required because they would never be used by mysql since we pass the openssl location in the build setup.